### DISCUSSION TITLE
Display Frage

### DISCUSSION THREAD

**Discussion by Da4force** [tbnobody/OpenDTU#936] [General]
Hallo,

erstmal tolles Projekt!

Habe gerade meine openDTU zusammengebaut und geflasht. Funktioniert alles soweit. Nur das Display ein SSD1306 zeigt die unteren pixel in Gelb. Ist das so normal oder muss noch was konfiguriert werden?

https://abload.de/img/pxl_20230519_14004964vuff1.jpg

**Comment by tbnobody** [comment]
Das liegt an deinem Display. Das ist zweifarbig. Da kann man auch nichts ändern

**Comment by Da4force** [comment]
Ah, danke für die info. Trotz Recherche nicht gesehen.
